### PR TITLE
feat(book): update CLI section

### PR DESCRIPTION
**Pull request by shekhirin**
`make update-book-cli` is a blessing

### CODE REVIEW

File: book/cli/reth/stage/run.md [hunk 1]
```diff
@@ -120,7 +132,7 @@ Networking:
           Do not persist peers.
 
       --nat <NAT>
-          NAT resolution method (any|none|upnp|publicip|extip:<IP>)
+          NAT resolution method (any|none|upnp|publicip|extip:\<IP\>)
```
Comment (DaniPopes) [inline book/cli/reth/stage/run.md:135]: Ah this is from the doc-comments

File: book/cli/reth/stage/run.md [hunk 2]
```diff
@@ -109,7 +121,7 @@ Networking:
       --identity <IDENTITY>
           Custom node identity
           
-          [default: reth/<VERSION>-<SHA>/<ARCH>]
+          [default: reth/<VERSION>-<SHA>/<ARCH>
```
Comment (DaniPopes) [inline book/cli/reth/stage/run.md:124]: need to stop at ] in the regex

**Review by DaniPopes — CHANGES_REQUESTED** [review]

File: book/cli/help.py [hunk 3]
```diff
@@ -254,8 +254,10 @@ def preprocess_help(s: str):
     """Preprocesses the help output of a command."""
     # Remove the user-specific paths.
     s = re.sub(r"default: /.*/reth", "default: <CACHE_DIR>", s)
-    # Remove the commit SHA.
-    s = re.sub(r"-[0-9A-Fa-f]{6,10}/", "-<SHA>/", s)
+    # Remove the OS
+    s = re.sub(r"default: reth/.*/(linux|macos|ios|freebsd|dragonfly|netbsd|openbsd|solaris|android|windows)", "default: reth/<VERSION>/<OS>", s)
+    # Remove the commit SHA and architecture
+    s = re.sub(r"default: reth/.*-[0-9A-Fa-f]{6,10}/.*", "default: reth/<VERSION>-<SHA>/<ARCH>", s)
```
Comment (DaniPopes) [inline book/cli/help.py:260]:
```suggestion
    s = re.sub(r"default: reth/.*-[0-9A-Fa-f]{6,10}/\w+", "default: reth/<VERSION>-<SHA>/<ARCH>", s)
```

File: book/cli/help.py [hunk 4]
```diff
@@ -254,8 +254,10 @@ def preprocess_help(s: str):
     """Preprocesses the help output of a command."""
     # Remove the user-specific paths.
     s = re.sub(r"default: /.*/reth", "default: <CACHE_DIR>", s)
-    # Remove the commit SHA.
-    s = re.sub(r"-[0-9A-Fa-f]{6,10}/", "-<SHA>/", s)
+    # Remove the OS
+    s = re.sub(r"default: reth/.*/(linux|macos|ios|freebsd|dragonfly|netbsd|openbsd|solaris|android|windows)", "default: reth/<VERSION>/<OS>", s)
```
[the remark this answers is not in the document]
Comment (shekhirin) [reply book/cli/help.py]: matching on triple now

**Review by rkrasiuk — APPROVED** [review]
indeed

**Review by mattsse — APPROVED** [review]

**Review by DaniPopes — APPROVED** [review]

**Pull request merged**